portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.data;
+
+/** A comment on a the main portfolio page */
+public final class Comment {
+    private final long id;
```

Comment:
Nice encapsulation. Private and final are good here. Be aware that you’ll need to add getters to allow these values to be retrieved.